content/setting-up-tugboat/select-repo-settings.md

```diff
@@ -127,6 +127,18 @@ credentials from within the repo settings.
 
 ![Authenticate with a Docker registry](../../_images/authenticate-with-a-docker-registry-add-credentials.png)
 
+By default, when you
+[specify a public image to use in Tugboat builds](/setting-up-services/how-to-set-up-services/specify-a-service-image),
+we use a Tugboat-authenticated user to pull images from Docker Hub.
```

Comment:
I can't put my finger on how to reword this, but we use our authentication to pull any image from Docker Hub, not just public images.